js/utility.js

```diff
@@ -22,4 +22,54 @@ export function unhideMultipleElements(array) {
     for (let i = 0; i < array.length; i++) {
         unhideElement(array[i]);
     }
-}
\ No newline at end of file
+}
+
+const levelPopUp = document.querySelector(".levelPopUp");
+
+function triggerPopUp(startWidth, endWidth, element) {
```

Comment:
you don't need an element here, replace element with levelPopUp you have already declared earlier
```suggestion
function triggerPopUp(startWidth, endWidth) {
///
levelPopUp.style.clipPath = `rect(0px ${currentWidth}% 100% 0px round 0%)`
```